src/mbgl/renderer/layers/render_circle_layer.cpp

```diff
@@ -71,10 +71,14 @@ void RenderCircleLayer::transition(const TransitionParameters& parameters) {
 }
 
 void RenderCircleLayer::evaluate(const PropertyEvaluationParameters& parameters) {
-    auto properties = makeMutable<CircleLayerProperties>(staticImmutableCast<CircleLayer::Impl>(baseImpl),
-                                                         unevaluated.evaluate(parameters));
+    const auto previousProperties = staticImmutableCast<CircleLayerProperties>(evaluatedProperties);
+    auto properties = makeMutable<CircleLayerProperties>(
+        staticImmutableCast<CircleLayer::Impl>(baseImpl),
+        unevaluated.evaluate(parameters, previousProperties->evaluated));
     const auto& evaluated = properties->evaluated;
 
+    // const auto y = unevaluated.getGPUExpressions(parameters.now);
```

Comment:
Commented out code.

Reply:
Just some leftovers to exercise all the template instances.